install/installer/pkg/components/server/configmap.go

```diff
@@ -74,6 +74,14 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil
 	})
 
+	chargebeeConfig := ""
```

Comment:
We've got rid of the default value for `ChargebeeProviderOptionsFile` here which was previously set to `"/chargebee/providerOptions"`, but since `EnablePayment` was always set to `false`, I don't think the value was ever actually used.

Reply:
My understanding is that this is a path on a volume (provided by the `chargebee-config` secret that we will create in the outer helm chart). We'll need to add the extra `volume` and `volumeMount` to the server deployment once we've added it.